fire-cracker/SendIT-react
# #165052316  User can track order

#### What does this PR do?
- Implements track orders 
- Implements track orders using actions

#### How should this be manually tested?
- Clone repo
- Check out to branch `feature/165052316/createOrder`
- Run application with `npm run start-dev` in the terminal

#### What are the relevant pivotal tracker stories?
- [Implement User can track order using react](https://www.pivotaltracker.com/story/show/165052316)
